NSS-Day-Cohort-66/rare-client-golden-droids-1
# All Categories List

Component at path `http://localhost:3000/categories` is now populated and there is a NavBar link to the path.

## Steps to Test

1. Pull branch `tp-view-all-categories`, switch to it, start the application
2. Make sure server side is updated to the latest and start debugger
3. Log in to application with the following:
```
username: phifertristan
password: phifer
```
4. Click the NavBar link `Category Manager` and confirm you are directed to `http://localhost:3000/categories`
5. While on this page, you should see the list of all categories. you can check your response code in Chrome dev tools or compare the data with your database file

closes #11
